client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -19,93 +20,117 @@ type OwnProps = {
 	expiryDate?: Moment;
 };
 
-const Paid: React.FC< OwnProps > = ( {
-	discountedPrice,
-	discountedPriceDuration,
+const Placeholder: React.FC< OwnProps > = ( { billingTerm, expiryDate } ) => {
```

Comment:
Moved the placeholder, discount price, and original price into their own components.